src/styles/rating-indicator.scss

```diff
@@ -228,10 +214,18 @@ $block: #{$fd-namespace}-rating-indicator;
     }
   }
 
-  &--half-star {
+  &.#{$block}--half-star {
     @include fd-rating-indicator-with-half-star($fd-rating-indicator-default-font-size, $fd-rating-indicator-default-spacing-between-icons);
   }
 
+  .#{$block}__label.#{$block}__zero-rating {
```

Comment:
This is violating BEM :) You can't have 2 elements from the same block at the same level. 
You can have:
```
<div fd-element__label fd-another-element__label>...</div>
```
to preserve the self-contained rules, but you can't have:

```
<div fd-element__label fd-element__zero-rating>...</div>
```

This can be achieved with modifier classes.